src/kvstore/raftex/RaftPart.cpp

```diff
@@ -324,7 +323,7 @@ void RaftPart::stop() {
     {
         std::unique_lock<std::mutex> lck(raftLock_);
         status_ = Status::STOPPED;
-        leader_ = {0, 0};
+        leader_ = {};
```

Comment:
How about `network::InetAddress(0, 0)`, it hard to tell what it is if we use `{}`;